CMakeLists.txt

```diff
@@ -27,6 +27,9 @@ print_var(PROJECT_VERSION)
 set(BUILD_TESTING OFF CACHE BOOL "Build with integration tests")
 print_var(BUILD_TESTING)
 
+set(BUILD_DOC OFF CACHE BOOL "Build doxygen documentation")
+print_var(BUILD_DOC)
```

Comment:
Maybe add this new option to this [documentation](https://github.com/JuPedSim/JuPedSim.github.io/blob/master/_posts/2016-11-02-QuickStart.md)

Reply:
I would like to update the doc last if possible i think some details might still change

Reply:
Maybe then move that file under `jpscore` at once.

Reply:
You mean move the build / dependency doc to the repos README?

Reply:
I meant in branch `gh-pages` under [_posts](https://github.com/JuPedSim/jpscore/tree/gh-pages/_posts)